postprocessing/land_utils/combine-ncc.F90

```diff
@@ -1,7 +1,26 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+!*
+!* FRE-NCTools is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FRE-NCTools (LICENSE.md) .  If not, see
+!* <http://www.gnu.org/licenses/>.
+!***********************************************************************
 !-----------------------------------------------------------------------
 ! Copyright 2011 NOAA Geophysical Fluid Dynamics Lab, Princeton, NJ
 ! This program is distributed under the terms of the GNU General Public
-! License. See the file COPYING contained in this directory
+! License.
 !
 ! This program reads several input netcdf files, presumably containing
 ! "compressed by gathering" data, and combines them into a single output file
```

Comment:
Please add a `(C)` to the copyright line after the word "Copyright".  

Remove the additional information about the license.  It is redundant (and in this case incorrect) as the header has the information.

Reply:
OK. Will do.

Reply:
Done.